tfx_addons/example_filter/data/test_data.csv

```diff
@@ -0,0 +1,51 @@
+label,col1
```

Comment:
Is this file identical to examples/example_filter/data/test_data.csv ?

Reply:
yes.I think that we need two identical files for the test case and the example case